app/services/hotkeys.ts

```diff
@@ -163,6 +163,20 @@ const SOURCE_ACTIONS: HotkeyGroup = {
     up: sourceId => getSourcesService().setMuted(sourceId, true),
     shouldApply: isAudio,
   },
+  HOTKEY_START: {
```

Comment:
Btw I keep meaning to ask, why are these called something so insanely generic as `HOTKEY_START`?  I assume this is what they're coming back from OBS as right?  Is this going to cause us issues in the future?  What if some other source is using a hotkey name this generic.  Do you think we should do anything about this now, since this is technically a breaking change for this hotkey anyway (this hotkey will become unbound for users when we release this).